homework2/reviewapp/src/app/all-shows/[id]/page.tsx

```diff
@@ -1,8 +1,16 @@
 'use client';
 
 import { ShowContainer } from "@/components/features/shows/ShowContainer/ShowContainer";
+import { AuthRedirect } from "@/components/shared/AuthRedirect/AuthRedirect";
 
 export default function ShowCardPage()
 {
-    return <ShowContainer/>
+
+    return (                
+        <>
+         {/* <AuthRedirect to="/login" condition="isLoggedOut"/> */}
+     <ShowContainer/> 
+        </>
```

Comment:
Why did you comment out the AuthRedirect here?

Reply:
When i was testing something i commented it out and forgot to change it back, I fixed it now